src/gwt/src/org/rstudio/studio/client/application/Application.java

```diff
@@ -249,14 +249,14 @@ else if (error.getCode() == ServerError.LICENSE_USAGE_LIMIT)
                // we display a special dialog for license limit issues
                // to allow the user to attempt to re-launch the session
                globalDisplay_.showGenericDialog(GlobalDisplay.MSG_ERROR,
-                                              "Licensing Limit Reached",
+                                              constants_.licensingLimitCaption(),
                                               error.getUserMessage() + "\n\n" +
-                                                 "Please quit any unused running sessions and try again, " +
-                                                 "or contact your administrator to update your license.",
+                                                 constants_.quitRunningSessionsMessage() +
+                                                 constants_.contactAdminMessage(),
                                               buttonLabels, elementIds, buttonOperations, 0);
             }
-            else if (StringUtil.equals(error.getUserMessage(), "Unable to connect to service") ||
-                     StringUtil.equals(error.getUserMessage(), "Error occurred during transmission"))
+            else if (StringUtil.equals(error.getUserMessage(), constants_.unableToConnectMessage()) ||
+                     StringUtil.equals(error.getUserMessage(), constants_.errorTransmissionMessage()))
```

Comment:
This code suggests logic that depends on exact string matches. Would need to do deeper review to see if this is a concern here. Leave this comment here so I remember to take another look.